src/components/vuestic-mixins/testIsSelectableList.js

```diff
@@ -0,0 +1,11 @@
+import { shallowMount } from '@vue/test-utils'
+
+// Ensure that component is proper form component
+export function testIsSelectableList (componentOptions) {
```

Comment:
Hmm. Not sure exactly how to improve, but that tests nothing right now.
Maybe you can test for testIsFormComponent here at least?

Reply:
I thought it can be just a placeholder for now. Later in components which uses this mixin we might want to test them correctly using it, so this helper might be useful? 

I'll add check `isFormComponent` to `SelectableListMixin` spec file though

Reply:
@misterpekert yeah, makes sense.